internal/mode/static/nginx/file/manager.go

```diff
@@ -88,7 +88,11 @@ func NewManagerImpl(logger logr.Logger, osFileManager OSFileManager) *ManagerImp
 func (m *ManagerImpl) ReplaceFiles(files []File) error {
```

Comment:
nit: better to make the commit message more descriptive and high level

`Add check to see if error is file does not exist`
->
`Fix NGF fails to recover if conf files are unexpectedly removed` (using the issue title here)

Reply:
Thanks, I was struggling to come up with a good commit message and pr title for this one